torch/csrc/jit/codegen/cuda/kernel_ir.h

```diff
@@ -1096,13 +1098,18 @@ class TORCH_CUDA_API GridReduction final : public Expr {
     return sync_buffer_;
   }
 
+  const ParallelTypeBitmap& thread_pred() const {
```

Comment:
Changed it to `threadPredicate` and added `setThreadPredicate`. There is `setPredicate`, so I used `setThreadPredicate` instead of `setThreadPred`. Perhaps, we should rename `setPredicate` so that the difference with thread predicates can be clear.